### ISSUE TITLE
Bunch of extra modules to download...

### ISSUE THREAD

**Issue by rde-girl** [radis/radis#680]
### 🐛 Describe the bug

Hello,
I installed radis on python 3.8 using the command: pip install radis
But then it's having me install a bunch of extra modules and I can't seem to reach the end.
What did I do wrong?
These are some of the modules it's having me install:
pip install numba
pip install pandas
pip install hjson
pip install termcolor
pip install h5py
pip install tables
pip instal joblib
pip install joblib
pip install astropy, etc.

### 🎲 Radis version

doesn't say

### 💻 Operating system

Windows

**Comment by minouHub** [comment]
Hello,
Thanks for raising this important issue!

### How to reproduce the bug
```
conda create -n issue680 python=3.8 -y #also tested with python3.10
conda activate issue 680
pip install radis
```
Then open python and type `import radis`. All the dependencies of radis are not installed!

### Quick fix
Install dependencies via radis=0.14, and then install the new version of radis.
```
pip install radis=0.14
pip install radis=0.15
```

Working on finding the issue. A fix will be proposed in radis 0.15.1

**Comment by minouHub** [comment]
Re-opening as `radis 0.15.1` is not merged yet. @rde-girl, hopefully `0.15.1` is online with corrections by next week

**Comment by rde-girl** [comment]
This solved it, thank you! But now I am running into another issue that I hadn't encountered before. I'm not sure if this is linked to the radis 0.15 version I downloaded. A similar was raised for radis 0.13 #482 

df = fetch_hitemp('H2O', load_wavenum_min=border_wing[0], load_wavenum_max=border_wing[1], isotope="1")

  File ~\.conda\envs\py38\lib\site-packages\radis\io\hitemp.py:148 in fetch_hitemp
    ldb.check_deprecated_files(

  File ~\.conda\envs\py38\lib\site-packages\radis\api\dbmanager.py:274 in check_deprecated_files
    check_not_deprecated(

  File ~\.conda\envs\py38\lib\site-packages\radis\api\cache_files.py:380 in check_not_deprecated
    file_metadata = manager.read_metadata(file)

  File ~\.conda\envs\py38\lib\site-packages\radis\api\hdf5.py:698 in read_metadata
    raise err

  File ~\.conda\envs\py38\lib\site-packages\radis\api\hdf5.py:695 in read_metadata
    metadata = f.get_storer(key).attrs.metadata

  File ~\.conda\envs\py38\lib\site-packages\pandas\io\pytables.py:1511 in get_storer
    raise KeyError(f"No object named {key} in the file")

KeyError: 'No object named df in the file'

**Comment by minouHub** [comment]
This occurs indeed when you switch from `vaex` DataFrame to a `pandas` DataFrame. The database has been saved to your disk in the `vaex` format but you try to load a `pandas` one.

As of August 2024, `vaex` is the standard if you're running with python 3.10 and below. The vaex team is currently working hard to upgrade their package (https://github.com/vaexio/vaex/pull/2331) and will be soon compatible with python 3.11 and 3.12.

### Quick fix

Delete the database file. Radis will automatically download a new one in the format you prefer. 

### What should be done in the long term. 

Provide a more explicit error message. If you would like to contribute, that would be a great first issue

**Comment by minouHub** [comment]
Hello,
This issue is solved with radis 0.15.1
Thanks again for reporting
